### ISSUE TITLE
"Choose scripts form scriptlog" input is not accessible using keyboard in the profile editor

### ISSUE THREAD

**Issue by WeirdRubberDuck** [OpenSpace/OpenSpace#3442]
Cannot select scripts from scriptlog using keyboard (Additional scripts). Even if I focus a item in the list with the mouse, I can’t move around using the up and down arrow keys

That is, this menu item (and the same when adding set property value from script log)

![image](https://github.com/user-attachments/assets/0b08ff06-44b8-4e43-b9b8-49234e09ab34)

**Comment by alexanderbock** [comment]
Check if this is actually an issue?

**Comment by WeirdRubberDuck** [comment]
Double checked, and yes, it is partly an issue - but not the same one as I thought initially. 

I can focus the scripts selection by tabbing, HOWEVER, the line in the log is not highlighted when focused. This is why I was confused (the only thing that's visually focused when tabbing is the "Save" button). 

If I press spacebar and use the up/down keys I can select brown in the edit. However, it's impossible to know which row will be selected before pressing spacebar. See video

https://github.com/user-attachments/assets/c4d4f9f2-d503-4a9b-873b-f113e1534af2

The contrast for the selected ones is also pretty low.
